### PR TITLE
use release when passed via cli

### PR DESCRIPTION
**Pull request by evgeni**
Fixes: #73

### CODE REVIEW

File: bootstrap.py [hunk 1]
```diff
@@ -977,6 +981,7 @@ def exec_service(service, command, failonerror=True):
         print "PARTITIONTABLE - %s" % options.partitiontable
         print "ORG - %s" % options.org
         print "ACTIVATIONKEY - %s" % options.activationkey
+        print "CONTENT RELEASE" % options.release
```
Comment (sideangleside) [inline bootstrap.py]: should be ` print "CONTENT RELEASE - %s" % options.release`

**Review by sideangleside — CHANGES_REQUESTED** [review]
Other than this one minor change, this LGTM.

File: bootstrap.py (hunk 1 above)
Comment (evgeni) [reply bootstrap.py]: ugh, right :face_with_head_bandage:

**Comment by evgeni** [comment]
fixed.

**Pull request merged**